src/ui/OverlayController.tsx

```diff
@@ -119,10 +137,13 @@ export class OverlayController extends React.Component<IControllerProps, void> {
           />
         </div>
         <div className="mdl-card__menu">
-          {remote.isEnabled ? <a onClick={toggleShareMenu}>SHARED</a> : null}
+          {remote.isEnabled ?
+            <a onClick={() => this.toggleShareMenu()}>SHARED</a>
+            : ""
+          }
           <button
             className="mdl-button mdl-button--icon mdl-js-button mdl-js-ripple-effect"
-            onClick={toggleShareMenu}
+            onClick={() => this.toggleShareMenu()}
```

Comment:
Just have `{ this.toggleShareMenu }`.

What you have currently creates a new anonymous function on every render (which is potentially detrimental to performance), and its only job is to call that method.  Your code should be simpler and faster if you get rid of the wrapping function.  😃